dockernet/ts-tests/test/main.test.ts

```diff
@@ -0,0 +1,180 @@
+import { Secp256k1HdWallet } from "@cosmjs/amino";
+import { GasPrice, SigningStargateClient } from "@cosmjs/stargate";
+import { fromSeconds } from "@cosmjs/tendermint-rpc";
```

Comment:
I remember you were in talks with the telescope team - do these packages compete with it or are these just lower level?

Reply:
these are low level helper functions, we can re-export them from stridejs instead of doing this

Reply:
No need to change anything - was just curious! I'm not really familiar with the ecosystem of JS packages